src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/StopPoints/Editor/Form.messages.js

```diff
@@ -58,4 +58,8 @@ export default defineMessages({
     id: 'lines.editor.deleteButtonText',
     defaultMessage: 'Slett',
   },
+  tooltip: {
+    id: 'lines.journeyPatterns.stopPoints.table.frontText.tooltip',
+    defaultMessage: 'Teksten som skal vises foran på bussen',
```

Comment:
Skal vi skrive `bussen`?

Reply:
vet ikke, kanske bara räcker med "...foran" ?

Reply:
Virker litt rart hvis man skal legge inn en ferge, og det står "..bussen" tenkte jeg bare 😛 kanskje "kjøretøyet" eller "transportmiddelet"? Kanskje julie har noen tanker

Reply:
Hehe ja, det er sant!

Reply:
Det er jo heller ingen kobling til at det som skrives inn der havner på forsiden av kjøretøyet..
Så det burde formuleres på en slik måte at man skjønner at det er for reiseplanleggingshensyn. Men har ikke noe godt forslag fra toppen av hodet mitt, hør med Julie.

Reply:
Hmm, det var vanskeligt med formuleringen. Vi får ta en runda på det sen!